server/main.go

```diff
@@ -162,11 +146,36 @@ func main() {
 		log.WithError(err).Fatal("failed to create authorizer.")
 	}
 
+	grpcUnaryInterceptors := []grpc.UnaryServerInterceptor{
+		grpc_ctxtags.UnaryServerInterceptor(),
+		grpc_logrus.UnaryServerInterceptor(logEntry),
+		grpc_prometheus.UnaryServerInterceptor,
+	}
+	grpcStreamInterceptors := []grpc.StreamServerInterceptor{
+		grpc_ctxtags.StreamServerInterceptor(),
+		grpc_logrus.StreamServerInterceptor(logEntry),
+		grpc_prometheus.StreamServerInterceptor,
+	}
+
 	if authorizer != nil {
```

Comment:
can authorizer ever be nil? Or shall we log warn or err when that happens or even panic?

Reply:
yes, it can be nil when user will specify no authorization at all (e.g just cert authorization)